tests/functional/test_llm_json.py

```diff
@@ -0,0 +1,362 @@
+#
+# Copyright (c) 2025 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+import os
+import logging
+
+model_name = os.getenv("MODEL_NAME", "meta-llama/Llama-3.1-8B-Instruct")
+base_url = os.getenv("BASE_URL", "http://localhost:8000/v3")
```

Comment:
can we use reservation manager?

Reply:
not yet, those tests can be executed manually, CI integration will be done later

Reply:
so for now this functionality has no unit tests or functional tests in CI :(